### PR TITLE
Telemetry: Handle failures determining disk usage

### PR DESCRIPTION
**Pull request by henrymercer**
We've noticed some situations where `df` fails due to the tracer not initializing correctly, and therefore we are unable to determine available disk space.  Let's instead fail gracefully when we can't determine available disk space.

### Merge / deployment checklist

- [x] Confirm this change is backwards compatible with existing workflows.
- [x] Confirm the [readme](https://github.com/github/codeql-action/blob/main/README.md) has been updated if necessary.
- [x] Confirm the [changelog](https://github.com/github/codeql-action/blob/main/CHANGELOG.md) has been updated if necessary.

### CODE REVIEW

File: src/status-report.ts [hunk 1]
```diff
@@ -241,6 +239,11 @@ export async function createStatusReportBase(
     workflow_run_id: workflowRunID,
   };
 
+  if (diskInfo) {
+    statusReport.runner_available_disk_space_bytes = diskInfo.numAvailableBytes;
+    statusReport.runner_total_disk_space_bytes = diskInfo.numTotalBytes;
```
Comment (aeisenberg) [inline src/status-report.ts:244]: Do we want to capture the reason for the `df` failure and send it as part of the status report?

**Review by aeisenberg — APPROVED** [review]
My comment shouldn't block having this merged, but something to add later if we want.

File: src/status-report.ts (hunk 1 above)
Comment (aeisenberg) [reply src/status-report.ts:244]: The message wouldn't make it into kusto, but at least it would be available in splunk.

Comment (henrymercer) [reply src/status-report.ts:244]: We could probably add an internal error diagnostic for this.  Let's address as potential follow up, as you suggest.

**Pull request merged**